### ISSUE TITLE
Backcountry -> Admin portal designs

### ISSUE THREAD

**Issue by Dianadec** [bcgov/reserve-rec-admin#1]
#### Description:

Begin to flesh out and design the admin portal for the new reservation service system. What do staff need? What tools? Reporting features? What would an admin page look like?

#### Development Checklist:

- [ ] Look at existing functionality in Camis
- [ ] Determine what is needed by staff for an admin page
- [ ] Mock-up initial designs
- [ ] Take out initial designs for feedback (note: this can be split into its own ticket)

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by LindsayMacfarlane** [comment]
Hi @pavelcreates  @meredithom - tagging you directly in this ticket for async backlog refinement. Please review criteria and add anything that is missing. Let me know if you have any questions.

**Comment by LindsayMacfarlane** [comment]
Apologies @pavelcreates - that was an incredibly bare-bones ticket! I've added some detail to the ticket/issue. Ultimately, what this ticket is looking for is some initial designs for an admin page in the reservation system. (Note: I hate the term portal - so feel free to change ;) )

cc @Dianadec

**Comment by LindsayMacfarlane** [comment]
This ticket is now a duplicate. Suggest closing/deleting. 

cc @Dianadec
